site/en/docs/web-platform/origin-trial-troubleshooting/index.md

```diff
@@ -167,13 +167,17 @@ For examples of pages that _do_ include an origin trial token, see the [demos](#
 
 ## Chrome DevTools status codes {: #devtools-status}
 
-* **Success**: The token is well-formed, has not expired, matches an origin trial feature, and is
-requested from an expected origin.<br>
-[Source code](https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/common/origin_trials/trial_token.cc;l=84)
+* **Expired**: Token has passed its expiration date. The token will need to be renewed, to
+generate a new token with a new expiration date.<br>
+[Source code](https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/common/origin_trials/trial_token_validator.cc;l=97)
 
-* **NotSupported**: The origin trial defined by the token is not supported in the Chromium
-'embedder': a browser such as Chrome or Edge, a WebView, or some other user agent.<br>
-[Source code](https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/common/origin_trials/trial_token_validator.cc;l=258)
+* **FeatureDisabled**: Trial is currently disabled for use.<br>
```

Comment:
```suggestion
* **FeatureDisabled**: Trial is currently disabled for all users.<br>
```

Reply:
@samdutton I'm unclear if this is true. If not, please clarify "use".

Reply:
Yeah — as you put it, the trial not available for anyone (any origin).